docs/tutorial/spark-tutorial/t-spark-history-server.md

```diff
@@ -0,0 +1,47 @@
+### Monitoring
+
+By default, Spark stores the logs of drivers and executors as pod logs in the local file system, which gets lost if the pods are deleted. Spark provides us with the feature to store these logs in S3 so that they can later be retrieved and visualized.
+
+Monitoring of Spark cluster can be done in two ways,
+1. Using Spark History server charm
+2. Using Canonical Observability Stack (COS)
+
+We'll cover monitoring with Spark history server in this section and cover monitoring using COS in the next section.
+
+### Monitoring with Spark History Server
+
+Spark History Server is is a user interface to monitor the metrics and performance of completed and running Spark applications. The Spark History Server is offered as a charm in the Charmed Spark solution, which can be deployed via Juju.
+
+To enable monitoring via Spark History server, we must first create a service account and provide configuration for Spark jobs to store logs in a S3 bucket. Then we need to deploy Spark History server with Juju by configuring it to read from the same S3 bucket that the Spark writes logs to.
+
+Since we already have an existing service account, let's add extra configurations related to monitoring and Spark History Server to the `spark` service account we created earlier.
```

Comment:
```suggestion
Since we already have the `spark` service account we created earlier, let's add extra configurations to instruct Spark History Server to save the logs in the S3 bucket using the `add-config` command. 
```